nvtabular/ops.py

```diff
@@ -1228,3 +1204,49 @@ def get_embedding_size(encoders, cat_names):
 
 def _emb_sz_rule(n_cat: int) -> int:
     return n_cat, int(min(16, round(1.6 * n_cat ** 0.56)))
+
+
+class LambdaOp(TransformOperator):
+    """
+    Enables to call Methods to cudf.Series
+
+    Parameters
+    -----------
+    op_name : str
+        name of the operator column. It is used as a post_fix for the
+        modified column names (if replace=False)
+    f : lambda function
+        defines the function executed on dataframe level, expectation is lambda col, gdf: ...
+        col is the cudf.Series defined by the context
+        gdf is the full cudf.DataFrame
+    columns :
+    preprocessing : bool, default True
+        Sets if this is a pre-processing operation or not
+    replace : bool, default True
+        Replaces the transformed column with the original input
+        if set Yes
+    """
+
+    default_in = ALL
+    default_out = ALL
+
+    def __init__(self, op_name=None, f=None, columns=None, preprocessing=True, replace=True):
+        super().__init__(columns=columns, preprocessing=preprocessing, replace=replace)
+        if op_name is None:
+            raise ValueError("op_name cannot be None. It is required for naming the column.")
+        if f is None:
+            raise ValueError("f cannot be None. DFLambda op applies f to dataframe")
```

Comment:
Should DFLambda be LambdaOp instead?

```suggestion
            raise ValueError("f cannot be None. LambdaOp applies f to dataframe")
```